infrastructure-playbooks/templates/logfile.j2

```diff
@@ -0,0 +1,14 @@
+Suggested cut and paste into "group_vars/osds.yml" for configuring with osd_scenario=lvm
```

Comment:
This file does not seem necessary, please address @guits's comment https://github.com/ceph/ceph-ansible/pull/2922#discussion_r207071829

Reply:
addressed!